video/out/mac/common.swift

```diff
@@ -395,7 +403,61 @@ class Common: NSObject {
     }
 
     func updateICCProfile() {
-        log.sendWarning("updateICCProfile not implemented")
+        guard let screenColorSpace = window?.screen?.colorSpace else {
+            log.sendWarning("Couldn't update ICC Profile, no color space available")
+            return 
+        }
+
+        colorSpace = screenColorSpace
+
+        if #available(macOS 10.11, *) {
+
+            // #HDR on #macOS        
+            var name = "" as CFString
+
+            switch (mpv?.macOpts.macos_output_csp)
+            {
+                case 3: name = CGColorSpace.itur_2020
+                        break;
+                default: break;
+            }
+
+#if HAVE_MACOS_10_12_FEATURES
+            if #available(macOS 10.12, *) {
+                switch (mpv?.macOpts.macos_output_csp)
+                {
+                    case 6: name = CGColorSpace.extendedSRGB; break
```

Comment:
breaks are not necessary on swift switch case. it will only fall through if an explicit fallthrough keyword is used.